media/js/firefox/new/scene1-fx-experiment.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var ua = navigator.userAgent;
+    var isFirefox = /\s(Firefox)/.test(ua) && !/Iceweasel|IceCat|SeaMonkey|Camino|like\ Firefox/i.test(ua);
+    var isDesktop = /\sFirefox/.test(ua) && !/Mobile|Tablet|Fennec|FxiOS/.test(ua);
+    var firefoxVersion = /Firefox\/(\d+(?:\.\d+){1,2})/.exec(ua);
+    var isUpToDate = firefoxVersion && firefoxVersion.hasOwnProperty(1) ? firefoxVersion[1].indexOf(62) === 0 : false;
```

Comment:
Nit: I believe `hasOwnProperty` is meant for objects as opposed to arrays. It seems to work here, but would checking the array length be a safer bet? 

```
firefoxVersion && firefoxVersion.length > 1
```